packages/amazonq/src/lsp/chat/messages.ts

```diff
@@ -188,6 +197,12 @@ export function registerMessageListeners(
                 }
                 break
             }
+            case OPEN_SETTINGS: {
+                const params = message.params as OpenSettingsParams
+                const setting = params.settingKey as keyof SettingsProps
```

Comment:
might as well be `as any`, since `as` means this doesn't actually achieve any type-checking. and that avoids needing to `export` SettingsProps.